frontend/src/v4/routes/components/dateField/dateField.component.tsx

```diff
@@ -14,12 +14,11 @@
  *  You should have received a copy of the GNU Affero General Public License
  *  along with this program.  If not, see <http://www.gnu.org/licenses/>.
  */
-import { ComponentType, useEffect } from 'react';
-import { DatePickerProps } from '@mui/lab/DatePicker';
-import { DateTimePickerProps } from '@mui/lab/DateTimePicker';
+import { useEffect } from 'react';
 import { TextField } from '@mui/material';
 import { useState } from 'react';
-import { StyledDatePicker, StyledDateTimePicker } from './dateField.styles';
+import { DateTimePicker } from '@controls/inputs/datePicker/dateTimePicker.component';
```

Comment:
- Now I can write after selecting a date inside the textbox
![stuff](https://github.com/3drepo/3drepo.io/assets/42032433/b4a0ca62-99e2-4d22-aad5-9fd67a1dc1ff)